pkg/health/client/client.go

```diff
@@ -317,8 +317,18 @@ func formatNodeStatus(w io.Writer, node *models.NodeStatus, printAll, succinct,
 	}
 	if succinct {
 		if printAll || !nodeIsHealthy(node) {
+			ips := []string{getPrimaryAddressIP(node)}
+			for _, addr := range GetHostSecondaryAddresses(node) {
+				var ip string
+				if addr == nil {
+					ip = "nil"
+				} else {
+					ip = addr.IP
+				}
+				ips = append(ips, ip)
+			}
 			fmt.Fprintf(w, "  %s%s\t%s\t%s\t%s\n", node.Name,
-				localStr, getPrimaryAddressIP(node),
+				localStr, strings.Join(ips, ","),
```

Comment:
If any address is somehow nil, then could the list show something like

```
1.1.1.1,nil,::1         reachable
```
?

If so, I think that's undesirable.

Reply:
Updated.